mxcube3/routes/log.py

```diff
@@ -8,7 +8,6 @@ def init_route(app, server, url_prefix):
 
     @bp.route("/", methods=["GET"])
     @server.restrict
-    @server.validate(tags=["log"])
```

Comment:
Should the `tags=["beamline"]` also be removed from [here](https://github.com/mxcube/mxcube3/blob/8b37bc56c17abd2dca30a777baf268e60aaca834/mxcube3/routes/beamline.py#L31)?

Reply:
Yes, indeed I missed that one :)